### PR TITLE
fix(almin): Do not call duplicated Store#onChange

### LINKED ISSUE #230: almin: StoreGroup#existWorkingUseCase is not working correctly
image

UnitOfWork commit to StoreGroup.
But, StoreGroup-self catch UseCase life-cycle event by `_observeDispatchedPayload`.
It cause timing issue I think.

### PR DESCRIPTION
**Pull request by azu**
fix #230

### CODE REVIEW

File: packages/almin/test/StoreGroup-edge-case-test.js [hunk 1]
```diff
@@ -42,4 +45,55 @@ describe("StoreGroup edge case", function() {
...
+                    callStack,
+                    [
+                        "change", // didExecute - receivePayload
+                        "did",
+                        "change", // complete - receivePayload
+                        "complete"
+                    ],
+                    "should be receivePayload -> almin lifecycle event"
+                );
+            });
```
Comment (azu) [inline packages/almin/test/StoreGroup-edge-case-test.js]: Test Case

File: packages/almin/src/Context.ts [hunk 2]
```diff
@@ -179,6 +179,7 @@ export class Context<T> {
             autoCommit: true
         });
         unitOfWork.open(useCaseExecutor);
+        useCaseExecutor.pipe(this._dispatcher);
```
Comment (azu) [inline packages/almin/src/Context.ts]: UseCase -> UseCaseExector -> { Unit Of Work } -> Dispatcher

Make it explicit.

- [ ] move this `pipe` in UnitOfWork?

**Comment by azu** [comment]
We should support `onBeginTransaction` and `onEndTransaction`.

**Comment by azu** [comment]
We should fix this image
https://github.com/almin/almin/blob/master/docs/resources/unit-of-work.png
It will be split up Read and Write.

**Comment by azu** [comment]
This change is not perfact.
We will continue to inverstigate.

[image](http://www.nomnoml.com/#view/%23padding%3A%2010%0A%5BUseCase%7C%0A%20%20%20%20%5BUseCase%20Executor%20A%7C%0A%20%20%20%20%20%20%20%20%5B%3Cusecase%3EChild%20UseCase%5D%0A%20%20%20%20%5D%20--%3E%20%5BUseCase%20Executor%20B%5D%0A%20%20%20%20%5BUseCase%20Executor%20B%7C%0A%20%20%20%20%20%20%20%20%5B%3Cusecase%3EUseCase%5D%0A%20%20%20%20%5D%0A%5D%0A%5BUnit%20of%20Work%7C%0A%09%5BStoreGroup%5D%0A%20%20%20%20%5BCommitments%7C%0A%20%20%20%20%20%20%20%20%5BDispatched%20Payload%5D%0A%20%20%20%20%20%20%20%20%5BSystem%20Payload%5D%0A%20%20%20%20%5D%20-%3E%20%5BStoreGroup%5D%0A%20%20%20%20%5BStoreGroup%7C%20%0A%20%20%20%20%20%20%20%20%5BStore%5D%0A%20%20%20%20%20%20%20%20%5B%3Cnote%3Eif%20any%20store%20is%20change%2C%20it%20emit%20changed%5D%0A%20%20%20%20%5D%0A%5D%0A%5BUseCase%5D%20%20--%3E%20%5B%3Creadonly%3EEventHub%5D%0A%5B%3Creadonly%3EEventHub%5D%20payload%20--%3E%20%5BContext%5D%0A%5BUseCase%5D%20payload%20--%3E%20%5BDispatcher%5D%0A%5BDispatcher%5D%20Payload--%3E%20%5BUnit%20of%20Work%5D%0A%5BUnit%20of%20Work%5D%20changes%20--%3E%20%5BContext%5D%0A)
